### PR TITLE
DataGrid - Adds new editing events to the "Row Editing and Editing Events" demo

### PR DESCRIPTION
**Pull request by NickMitrokhin**

### CODE REVIEW

File: NetCoreDemos/DevExtreme.NETCore.Demos.csproj [hunk 1]
```diff
@@ -24,7 +24,7 @@
     <PackageReference Include="Microsoft.Azure.Storage.Blob" Version="11.1.0" />
     <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="2.1.11" />
     <PackageReference Include="DevExtreme.AspNet.Data" Version="2.6.1" />
-    <PackageReference Include="DevExtreme.AspNet.Core" Version="20.2.0" />
+    <PackageReference Include="DevExtreme.AspNet.Core" Version="20.2.2" />
```
Comment (vconst) [inline NetCoreDemos/DevExtreme.NETCore.Demos.csproj]: why?

**Review by vconst — CHANGES_REQUESTED** [review]

**Review by vconst — COMMENTED** [review]
Add updated tag to the demo in menuMeta.json

File: NetCoreDemos/DevExtreme.NETCore.Demos.csproj (hunk 1 above)
Comment (NickMitrokhin) [reply NetCoreDemos/DevExtreme.NETCore.Demos.csproj]: It was modified automatically when all preparation scripts were executed.

Comment (NickMitrokhin) [reply NetCoreDemos/DevExtreme.NETCore.Demos.csproj]: I rolled back this change.

**Review by vconst — APPROVED** [review]

**Pull request merged**